pulp_rpm/app/serializers.py

```diff
@@ -704,3 +705,25 @@ class Meta:
             "base_product_version", "arch", "build_timestamp", "instimage", "mainimage",
             "discnum", "totaldiscs", "addons", "checksums", "images", "variants"
         )
+
+
+class RepoMetadataFileSerializer(SingleArtifactContentSerializer):
+    """
+    RepoMetadataFile serializer.
+    """
+
+    data_type = serializers.CharField(
+        help_text=_("File type.")
```

Comment:
Maybe "metadata type" is a better name, what to you think?
When I read "file type", I think more about file formats, like jpeg, pdf, etc.